src/layout/header/Header.tsx

```diff
@@ -155,14 +174,24 @@ export const Header = () => {
                 </MenuButton>
               )}
               {(hasCuratorRole(user) || user?.isNviCurator) && (
-                <MenuButton
-                  color="inherit"
-                  data-testid={dataTestId.header.tasksLink}
-                  isSelected={currentPath.startsWith(UrlPathTemplate.Tasks)}
-                  to={UrlPathTemplate.Tasks}
-                  startIcon={<AssignmentIcon />}>
-                  {t('common.tasks')}
-                </MenuButton>
+                <Badge
+                  badgeContent={notificationCounts || null}
```

Comment:
skal badge også vises på mobil-meny?

Reply:
Har lyst til å utsette, for å ha noe klart til ekstern demo i morgen. Notifications skal også inn på min side, så blir flere oppgaver på dette.